agent/engine/serviceconnect/manager_linux.go

```diff
@@ -391,9 +412,14 @@ func (agent *manager) IsLoaded(dockerClient dockerapi.DockerClient) (bool, error
 }
 
 func (agent *manager) GetLoadedImageName() (string, error) {
```

Comment:
Should we remove returning `error` from the return value since it will always be nil?

Reply:
we return error for unsupported platforms so I think good to keep it